tests/console/yast2_clone_system.pm

```diff
@@ -42,24 +42,20 @@ sub run {
     # Check and upload profile for chained tests
     upload_asset $ay_profile_path;
 
-    unless (is_opensuse) {
+    unless (is_opensuse || is_staging) {
```

Comment:
this block has no purpose now as you always use jing from Leap not from these repos. @rwx788 WDYT? Should we still use sle repos for sle?

Reply:
We could apply this workaround solution only in case package is not available in the system. In general I'm not a fan of hardcoded urls. And if we use one, I would prefer more static ones and not the one which will stop working once there is a new version available.
So TBH, I would simply add Factory repo, install jing from it and remove it after the test, not to pollute the setup.

Reply:
There is a problem to use `CURRENT` e.g. `ftp://openqa.suse.de/SLE-12-SP5-WE-POOL-x86_64-CURRENT-Media1/`, that got broken recently and seems there is no patch expected.

Reply:
i cant see development-tools with current anyway(ftp://openqa.suse.de/). I have tried to use another repo but I still have the gpg verification problem. if someone show me i can try again because i miss something talking about Factory

Reply:
So i tried a few times the approach with http://download.opensuse.org/tumbleweed/repo/oss/ to find out that the jing is not provided neither there. So i think we are kinda stuck with this 'ungly' solution for now I have addressed the nice catch from @jknphy and provide VR for those cases. RFR